dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -4943,8 +4973,23 @@ int ha_mcs_impl_pushdown_init(mcs_handler_info* handler_info, TABLE* table, bool
     ByteStream msg;
     ByteStream emsgBs;
 
+    int ntries = 10;
+
+    // The issue is MCOL-5396.
+    // The delay below is used to trigger old infinite loop condition and
+    // prove that mitigation works. While it looks like unused code, it is
+    // important enough to have it just in case.
+    // using namespace std::chrono_literals;
+    // std::this_thread::sleep_for(10000ms);
```

Comment:
please remove unused code

Reply:
The code is not unused - it is part of comment on how to reproduce the condition. Thus, I would like to have it there.

Reply:
You already put the note that you need sleep and then you add 2 extra lines: one with the code itself and another to explain this codeline is needed. Isn't it good enough to mention sleep and avoid the specific sleep syntax. What if the sleep syntax will change overtime for example?

Reply:
I do my best to lower the workload of the (future) colleagues. In this particular case I relieve them from looking into documentation, this saves a minute or two, yet it saves.

The purpose is same or close to commented code next to FIXME in ha_mcs_execplan.cpp. To have code ready to be used yet disabled.

As for change of syntax, this is standard from C++11 and most probably set in stone.